auth-foundation/src/main/java/com/okta/authfoundation/jwt/Jwks.kt

```diff
@@ -56,12 +56,12 @@ class Jwks internal constructor(
 internal class SerializableJwks(@SerialName("keys") val keys: List<Key>) {
     @Serializable
     class Key(
-        @SerialName("alg") val algorithm: String,
-        @SerialName("e") val exponent: String?,
-        @SerialName("n") val modulus: String?,
         @SerialName("kid") val keyId: String,
-        @SerialName("kty") val keyType: String,
         @SerialName("use") val use: String,
+        @SerialName("kty") val keyType: String,
+        @SerialName("alg") val algorithm: String? = if (keyType == "RSA") "RS256" else null,
```

Comment:
Does this happen with some authorization servers? i.e. the `alg` is omitted?

Reply:
Yes, I've seen it in 2 now.